src/templates/index.html

```diff
@@ -120,31 +121,13 @@ <h2 class="w-300 color-dark mb-10">To add transparency to the nonprofit sector</
 
 
 			<div class="row mb-60 xs-mb-20">
-				<div class="col-sm-1 col-lg-offset-1">
-					<i class="icon icon-globe"></i>
-				</div>
-				<div class="col-sm-4 col-md-3">
-					<h3 class="mb-15">Worldwide</h3>
-					<p class="smaller xs-mw">Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.
-				</div>
-			</div>
-
-			<div class="row mb-60 xs-mb-20">
-				<div class="col-sm-1 col-lg-offset-1">
-					<i class="icon icon-eye"></i>
-				</div>
-				<div class="col-sm-4 col-md-3">
-					<h3 class="mb-15">Privacy</h3>
-					<p class="smaller xs-mw">Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.
-				</div>
-			</div>
-			<div class="row">
-				<div class="col-sm-1 col-lg-offset-1">
-					<i class="icon icon-bubble"></i>
+				<div class="col-sm-1">
 				</div>
-				<div class="col-sm-4 col-md-3">
-					<h3 class="mb-15">Social Integration</h3>
-					<p class="smaller xs-mw">Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.
+				<div class="mission-text col-sm-4 col-md-5">
+					<p class="para">The nonprofit sector currently stands at about 5% or so of US GDP and hasn’t experienced any profound growth over the past several years. Donations are being scattered to many firms that support the same initiatives and firms are forced to squeeze their overhead costs until they are hardly able to operate (let alone expand their operations).</p><br>
```

Comment:
We can probably add the more information of the score here, such as the importance, scale, and algorithm.